h/htslib/build_info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "htslib",
+    "github_url": "https://github.com/samtools/htslib",
+    "version": "1.17",
+    "default_branch": "develop",
+    "package_dir": "h/htslib/",
+    "build_script": "htslib_ubi_8.5.sh",
+    "validate_build_script": "true",
+    "use_non_root_user": "false",
+    "docker_build": false
+}
```

Comment:
Please see if we can continue to support old script for old version by adding regex in build_info.json